<?xml version="1.0"?>
<launch>

  <arg name="robot" default="holonomic"/>
  <arg name="paused" default="false"/>
  <arg name="auto_quit" default="true"/>
  <arg name="use_sim_time" default="true"/>
  <arg name="gui" default="true"/>


 <!-- Launch PCL obstacle tracker -->
  <node pkg='kf_tracker_2' type='kf_tracker_2_node' name='kf_tracker_2' output="screen">
    <param name='frequency' value='10.0'/>

    <param name='logSigma2_x' value='-5.5'/>
    <param name='logMagnSigma2_x' value='-3.5'/>
    <param name='logLengthScale_x' value='0.75'/>

    <param name='logSigma2_y' value='-5.5'/> 
    <param name='logMagnSigma2_y' value='-3.5'/> 
    <param name='logLengthScale_y' value='0.75'/> 

    <param name='data_length' value='40'/>
    <param name='param_fix' value='true'/>

  </node>

</launch>
